specification/Overview.md

```diff
@@ -9,18 +9,53 @@ organised by data category (saveframe).
 
 ### General issues
 
-  1. Residue and Atom names
-
-    Residues are identified by three strings ('chain_code', 'sequence_code',
-    'residue_type') that always are used together. Atoms are identified by a
-    residue identifier plus the atom name as a fourth string.
+  1. Requirements for NEF compatibility
+
+    The NEF format tries to support as many kinds of data as possible in a
+    simple file format. But we can clearly not require any program to
+    read and understand data that it does not normally support, be it complex or
+    crosslinked sequences, non-standard residues or residue variants, OR/AND
+    restraint logic, anonymous atom assignments, particular restraint types,
+    etc. If a program does support a certain type of data,
+    it is required to read and write such data correctly to/from NEF. If it
+    does not, it is free to skip such data, or convert them to something it can
+    deal with if this can be done appropriately.
```

Comment:
If for example you pass a complete NEF project, with restraints etc. to TALOS, I would expect to get back the sequence, the shift list used as input, the resulting dihedrals - and that is pretty much it.

The original prooposal was to have all programs retain the full file, as you also suggest, and that would clearly have been much better. But the progam developers told us that they could not undertake to track and re-export data that they did not use and that their program could not interpret. And since the program developers are the customers, they are pretty much always right.